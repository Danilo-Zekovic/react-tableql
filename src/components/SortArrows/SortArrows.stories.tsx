import React from 'react'

import SortArrows from './SortArrows'
import { STORYBOOK_SECTIONS } from '../../../stories/constants'

export default {
  title: `${STORYBOOK_SECTIONS.buildingBlocks}|SortArrows`,
  component: SortArrows,
}

export const Default = () => <SortArrows />
