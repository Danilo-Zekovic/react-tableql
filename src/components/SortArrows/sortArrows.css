.TableQL-thead-th-sort-arrows {
  float: right;
  display: flex;
  flex-direction: column;
  height: 100%;
  align-items: center;
}

.TableQL-thead-th-sort-arrow-up {
  width: 0;
  height: 0;
  border-left: 5px solid transparent;
  border-right: 5px solid transparent;
  border-top: 5px solid transparent;

  border-bottom: 5px solid #787C80;
  margin-bottom: 1px;
}

.TableQL-thead-th-sort-arrow-down {
  width: 0;
  height: 0;
  border-left: 5px solid transparent;
  border-right: 5px solid transparent;

  border-top: 5px solid #787C80;
  margin-top: 1px;
}
