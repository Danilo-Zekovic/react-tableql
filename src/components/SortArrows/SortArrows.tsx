import React from 'react'

import './sortArrows.css'

const SortArrows = () => (
  <div className="TableQL-thead-th-sort-arrows">
  <div className="TableQL-thead-th-sort-arrow-up" />
  <div className="TableQL-thead-th-sort-arrow-down" />
  </div>
)

export default SortArrows
