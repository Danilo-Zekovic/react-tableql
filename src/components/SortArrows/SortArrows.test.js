import React from 'react'

import { render } from '@testing-library/react'

import SortArrows from './SortArrows'

describe('<SortArrows>', () => {
  it('default snapshot', () => {
    const { container } = render(<SortArrows />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
