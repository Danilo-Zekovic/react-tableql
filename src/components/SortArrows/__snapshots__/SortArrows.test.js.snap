// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SortArrows> default snapshot 1`] = `
<div
  class="TableQL-thead-th-sort-arrows"
>
  <div
    class="TableQL-thead-th-sort-arrow-up"
  />
  <div
    class="TableQL-thead-th-sort-arrow-down"
  />
</div>
`;
